Files/ViewModels/Dialogs/DynamicDialogViewModel.cs

```diff
@@ -219,16 +385,19 @@ public DynamicDialogViewModel()
         {
             // Create default implementation
             TitleText = "DynamicDialog";
-            PrimaryButtonText = "OK";
+            PrimaryButtonText = "Ok";
```

Comment:
I guess it is should be translated.

Reply:
It's a fallback value and should always be assigned when creating the dialog. This value should not be used, and I've only kept it here so the button is visible